src/it/streamingcommunity/src/eu/kanade/tachiyomi/animeextension/it/streamingcommunity/StreamingCommunity.kt

```diff
@@ -344,13 +360,33 @@ class StreamingCommunity : ConfigurableAnimeSource, AnimeHttpSource() {
     }
 
     companion object {
+        private const val PREF_DOMAIN_KEY = "referred_domain"
+        private const val PREF_DOMAIN_TITLE = "Override BaseUrl"
+        private const val PREF_DOMAIN_DEFAULT = "https://streamingcommunity.forum"
+        private const val PREF_DOMAIN_SUMMARY = "For temporary uses. Updating the extension will erase this setting."
```

Comment:
The problem is that this preference doesn't actually get erased on a new extension update, and it really should. Unless you can find a way to do that, I'd advise against adding in such an option. (Also, don't rely on buildconfig for that)

Reply:
It would only be temporary since this website changes domains like changing shoes

Reply:
The problem is that the current implementation isn't temporary, and the url won't be updated on an extension update (which it should)